README.md

```diff
@@ -28,6 +28,23 @@ Then copy the `./bin/swctl-latest-(darwin|linux|windows)-amd64` to your `PATH` d
 usually `/usr/bin/` or `/usr/local/bin`, or you can copy it to any directory you like,
 and add that directory to `PATH`, we recommend you to rename the `swctl-latest-(darwin|linux|windows)-amd64` to `swctl`.
 
+In addition, you can also quickly install through the following methods:
+
+**Install swctl on Linux**
+
+```shell
+sudo ./scripts/install.sh
```

Comment:
You should assume that users don't clone the project, so the command should be like this: `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`